src/slack/publishMessages.js

```diff
@@ -1,10 +1,17 @@
 const { postMessage } = require('./chat');
 const composeBlocks = require('./composeBlocks');
 
-module.exports = async ({ channel, repository, branches }) => {
+const normalHeaderMessage = ({ branches, repository }) =>
+  `<!here> There are *${branches.length}* stale branches in *${repository}*! Click the delete button if the branch is no longer used. :do_not_litter:`;
+const zeroStaleHeaderMessage = ({ repository }) =>
+  `🎊 There are *ZERO* stale branches in *${repository}*!👏`;
+
+const defaultFunction = async ({ channel, repository, branches }) => {
```

Comment:
When there are 0 stale branches, it would be more ideal not to include the divider block (line 23) because there will be no following messages.

![image](https://user-images.githubusercontent.com/5466341/95032116-63d22480-0687-11eb-9274-f131b739c96a.png)

Reply:
👍 updated